web-app/ui/src/wrapper/AuthWrapper.js

```diff
@@ -0,0 +1,12 @@
+import { Navigate, Outlet } from "react-router-dom";
+import {SessionStore} from "services/store";
+
+export const AuthWrapper = () => {
+  const user = SessionStore.getUser();
+
```

Comment:
yahan pe sessionstore ki jagah auth context use kar skta

Reply:
Hn lekin, but auth context is not accessible in loader where I need to fetch current user objects using id from the user